@stellar/typescript-wallet-sdk/package.json

```diff
@@ -69,6 +70,7 @@
     "build:web": "webpack --config webpack.config.js",
     "build:node": "webpack --env NODE=true --config webpack.config.js",
     "build": "run-p build:web build:node",
+    "example:sep10": "ts-node examples/sep10/sep10Server.ts",
```

Comment:
how should we run`sep10Wallet` ?

Reply:
Ah good call. I wasn't sure if this was useful to run as it's a simple helper, but I can add it!